Update JDBC trace for Oracle 23c (#7415)
OpenLiberty/docs: modules/ROOT/pages/jdbc-tracing.adoc

[#Oracle]
=== Oracle

> Prior to Oracle 23c Oracle provided two different drivers, one for production and another for debugging purposes. The production driver does not produce any trace, so you need to download and replace your production driver with the debugging driver. The debugging driver has `_g` in the driver name. For example, `ojdbc8.jar` is `ojdbc8_g.jar`. Configure the debugging driver by specifying the `library` subelement within the `jdbcDriver` element. The `library` subelement defines the path to the debugging driver JAR file.

Reviewer: ```suggestion
Prior to Oracle 23c, Oracle provided two different drivers, one for production and another for debugging purposes. The production driver does not produce any trace, so you need to download and replace your production driver with the debugging driver. The debugging driver has `_g` in the driver name. For example, `ojdbc8.jar` is `ojdbc8_g.jar`. Configure the debugging driver by specifying the `library` subelement within the `jdbcDriver` element. The `library` subelement defines the path to the debugging driver JAR file.
```

> After Oracle 23c there are no `_g` drivers and diagnotic trace is provided in the base JDBC driver.

Reviewer: ```suggestion
In Oracle 23c and later, Oracle no longer provides the `_g` drivers and diagnostic trace is provided in the base JDBC driver.
```
